lib/v2/components/text_form_field_light_custom.dart

```diff
@@ -0,0 +1,90 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:seeds/constants/app_colors.dart';
+import 'package:seeds/design/app_theme.dart';
+
+/// A widget wrapper of TextFormField customized for general inputs
```

Comment:
dont we have a custom input text already? 
@RaulUrtecho

Reply:
What happens is that I told Gabriel that the best thing would be to have two.
Because of the light theme and dark theme. 
I thought it would be easier to have 2 custom widgets since there are no more types of Textfield only these and to avoid to load with extra logic the current widget to also check if it is litght theme or dark theme.